lms/djangoapps/commerce/models.py

```diff
@@ -15,6 +15,7 @@ class Meta(object):
 
     API_NAME = 'commerce'
     CACHE_KEY = 'commerce.api.data'
+    DEFAULT_RECEIPT_PAGE_URL = '/commerce/checkout/receipt/?orderNum='
```

Comment:
That's only a default value that's set if the admin user doesn't provide a value there. I can change the default value if needed.

Admin user needs to go to the Commerce Configuration and set the receipt page relative URL. See screenshot no.1.